nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentUtils.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.platform.comment;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.nuxeo.ecm.platform.comment.api.CommentConstants.COMMENT;
+import static org.nuxeo.ecm.platform.comment.api.CommentConstants.COMMENT_DOCUMENT;
+import static org.nuxeo.ecm.platform.comment.api.CommentConstants.PARENT_COMMENT;
+import static org.nuxeo.ecm.platform.comment.workflow.utils.CommentsConstants.COMMENT_TEXT;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.event.Event;
+
+/**
+ * @since 11.1
+ */
+public class CommentUtils {
+
+    // Suppress default constructor for non instantiability
+    private CommentUtils() {
+        throw new AssertionError();
```

Comment:
just a comment would be enough

Reply:
yes i agree, it is just a way to avoid any inner classe instantiation.  i mean under this `CommentUtils` wdyt

Reply:
Personnaly I think it's overkill for such a test class that will probably never be extended, and even in the case of an inner class, the AssertionError wouldn't be clear.
But this is just a detail!